specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/ContainerApps.json

```diff
@@ -856,6 +860,36 @@
         }
       }
     },
+    "Runtime": {
+      "description": "Container App Runtime configuration.",
+      "type": "object",
+      "properties": {
```

Comment:
`kind` and `java` should not have default values, because the user may only want to update one of them.
https://github.com/Azure/azure-rest-api-specs/pull/27892/checks?check_run_id=22018340669

Reply:
thanks, default value removed